doc/reference/devices_disk.md

```diff
@@ -70,6 +70,20 @@ VM `cloud-init`
 
       lxc config device add <instance_name> <device_name> disk source=cloud-init:config
 
+## Initial Configuration for Instance Root Disk Devices
+
+Initial configuration allows setting specific configurations for the root disk devices of new instances.
+These settings are prefixed with `initial.` and are only applied when the instance is created.
+This enables the creation of instances with unique configurations, independent of the default storage pool settings.
+
+For example, initial configuration for `block.filesystem` can be set to an existing profile, taking effect for each new instance created:
```

Comment:
```suggestion
For example, you can add an initial configuration for `block.filesystem` to an existing profile, and this will then take effect for each new instance you create using this profile:
```